docs/operator/configuration_parameters.md

```diff
@@ -532,36 +532,48 @@ and only contains information about the specific RSE's for the Rucio instance.
 Read more about RSEs [here](../started/concepts/rucio_storage_element.md)
 and how to set them up [here](./configuration.md#creating-new-rses).
 
-- **associated_sites**: <!--??--> Separated by commas. Default: `None`.
-- **auto_approve_bytes**: Auto Approve Limit. Example: `500GB`. No default.
+- **associated_sites**: <!--??--> String. Separated by commas. Used for chaining of subscriptions so that transfers to one RSE will also be mirrored to associated_sites. No default.
+- **archive_timeout**: <!--??--> Integer: For transfers with a tape destination, control the number of seconds Rucio will wait for the tape archival of the file to complete. No default.
+- **auto_approve_bytes**: <!--??--> Auto Approve Limit. Example: `500GB`. No default.
```

Comment:
Is this value under which transfers are automatically approved? Worth making that explicit. Same for `auto-approve-files`